R/principal_components.R

```diff
@@ -419,14 +419,8 @@ principal_components.data.frame <- function(x,
     n <- ncol(x) - 1
   } else if (n >= ncol(x)) {
     n <- ncol(x) - 1
```

Comment:
I would say this should actually be setting `n <- ncol(x)`. Plausibly someone could want to just transform variables into principal components without any dimension reduction 

@strengejacke